### PR TITLE
Test: Resource looker_user_attribute_user

### PR DESCRIPTION
**Pull request by Vitzeno**
# Description

Adds tests for the `looker_user_attribute_user` resource, including API checks

### CODE REVIEW

File: looker/resource_looker_user_attribute_user_test.go [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+			}, nil)
+			if err != nil {
+				return err
+			}
+
+			for _, u := range users {
+				if _, err := c.DeleteUser(*u.Id, nil); err != nil {
+					return err
+				}
+			}
```
Comment (ponteilla) [inline looker/resource_looker_user_attribute_user_test.go:57]: We already have a sweeper for similar users. Could we maybe use the `Dependencies` slice attribute on `resource.Sweeper`?

File: looker/resource_looker_user_attribute_user_test.go [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+			UserId: userRes.Primary.ID,
+			Fields: conv.P(""),
+		}, nil)
+		if err != nil {
+			return errors.Wrapf(err, "failed to retrieve group with id: %v", userRes.Primary.ID)
+		}
+
+		userIds := []string{}
+		for _, userAttr := range userAttrs {
+			userIds = append(userIds, *userAttr.UserId)
```
Comment (ponteilla) [inline looker/resource_looker_user_attribute_user_test.go:125]: I'm wondering if we could pass the user attribute resource to the `TestCheckFunc` and the value as well to check it's been set to `25`?

Maybe overkill.

Would it simplify inspecting the request made above by adding something like `UserAttributeIds: &rtl.DelimString{"attrID"}` to it?

So something like:
```golang
if userAttr.Name == attrName && userAttr.Value == attrValue {
  return nil
}
```
Then default to returning an error?

File: looker/resource_looker_user_attribute_user_test.go (hunk 1 above)
Comment (Vitzeno) [reply looker/resource_looker_user_attribute_user_test.go:57]: That's a very useful resource and works really well, might have to back and check if any of the other tests can benefit from this too.

File: looker/resource_looker_user_attribute_user_test.go (hunk 2 above)
Comment (Vitzeno) [reply looker/resource_looker_user_attribute_user_test.go:86]: I'm not sure that would work, the `looker_user_attribute` resource sets a default value for all users and the `looker_user_attribute_user` resources overrides that with a custom values for the current user, they'll always be different in this case.

I can check the value set by the `looker_user_attribute_user` resource and the default value set by the `looker_user_attribute` resource in its own test instead?

**Review by ponteilla — APPROVED** [review]

**Pull request merged**